### PR TITLE
Pulled up themes, changed navbar and about to be mui components

### PR DESCRIPTION
**Pull request by JeromeTing**
image

- FIxed the about page, but container xl doesn't fit all the cards :( I can change it to false and it'll just fit the entire width.
- Navbar and about page are MUI components using global theme
- changed all components to use themes 
- changed scoreboard imports from material-ui to mui (this allows the use of global themes)

### CODE REVIEW

File: src/components/AboutUsCard.js [hunk 1]
```diff
@@ -52,8 +53,9 @@ export default function AboutUsCard({developer}) {
     }));
 
     return (
-        <div className="about-us-card">
-        <Card sx={{ display: 'flex' , background: 'darkgray', width: 'fit-content' }}>
+        <ThemeProvider theme={theme}>
```
Comment (kernelkay) [inline src/components/AboutUsCard.js]: Couldn't we just wrap the `App` component with `ThemeProvider` so we only have to add it once?

**Review by kernelkay — CHANGES_REQUESTED** [review]

File: src/components/AboutUsCard.js (hunk 1 above)
Comment (JeromeTing) [reply src/components/AboutUsCard.js]: You're right, I just pushed the new changes

**Review by kernelkay — CHANGES_REQUESTED** [review]
hmm seems the styling messed up the search bar


Screen Shot 2023-06-07 at 10 20 19 PM

Screen Shot 2023-06-07 at 10 20 10 PM

**Comment by JeromeTing** [comment]
> hmm seems the styling messed up the search bar
> 
> Screen Shot 2023-06-07 at 10 20 19 PM Screen Shot 2023-06-07 at 10 20 10 PM

Apparently the reason the outline went white was because the theme was dark and our background is white. I've wrapped that portion of the search using a light theme so we can see the outline again

image
image

**Review by Parmvir-S — APPROVED** [review]

**Review by kernelkay — APPROVED** [review]

**Pull request merged**